neofetch

```diff
@@ -950,6 +950,10 @@ get_distro() {
                     *) distro="Red Star OS $(awk -F'[^0-9*]' '$0=$2' /etc/redstar-release)"
                 esac
 
+            elif [[ -f /etc/armbian-release ]]; then
+                . /etc/armbian-release
+                distro="Armbian $DISTRIBUTION_CODENAME ($VERSION)"
+
```

Comment:
None of the existing detection methods work?

Reply:
`cat /etc/os-release` or `lsb-release -sir`?

Reply:
For compatibility reasons /etc/os-release remains Debian or Ubuntu.